controller/controller.go

```diff
@@ -260,7 +261,7 @@ func (c *Controller) bootWithError(ctx context.Context) error {
 		o := manager.Options{
 			// MetricsBindAddress is set to 0 in order to disable it. We do this
 			// ourselves.
-			MetricsBindAddress: "0",
+			MetricsBindAddress: DisableMetricsServing,
```

Comment:
It's not just about re-using the same value thanks to the constant, but also making the code easier to read. With the constant, I can see that we are disabling serving metrics. Using the '0' I need to know what '0' means in this context.
Are there any negative trade-offs using a constant?

Reply:
I would argue it's harder to read now because I need to find the const and figure what's the value behind that. The comment already explains it well. I understand it's easy with an IDE but I think about the browser and reviews. Let's not waste time on it though. I can disagree and commit.